lib/resources/user.js

```diff
@@ -15,10 +15,7 @@ module.exports = BaseResource.extend({
     developer: 'boolean',
     twitter: 'string'
   },
-  login: function(callback) {
-    var options = {
-      userId: this.userId
-    };
+  login: function(options, callback) {
```

Comment:
@mikhail-hatsilau is this a backwards breaking change? It looks like the API is modified and now requires to pass the `userId` in via options? If that's the case, we'll need to bump the major version of the npm package.

Reply:
@rueckstiess It is required. We have userId field inside the resource itself. When we call login, userId from the resource saves into db. Having userId in the options seems to be duplication